### PR TITLE
Banner Pages

### PR DESCRIPTION
**Pull request by erenfn**
image

Test it by http://localhost:5173/banner
- Removed Box wrapper in CustomTextField
- Implemented the Create Banner Page with functionality
- Implemented GuideTemplate so that other people can use it to create other similar pages

Need Changes to:
- Radio Component (Kenan) 
- DropdownList (Rusi) - just some styling

### CODE REVIEW

**Comment by erenfn** [comment]
- Also removed login functionality for testing purposes

**Review by uparkalau — COMMENTED** [review]
@coderabbitai review

**Review by uparkalau — COMMENTED** [review]
@coderabbitai review the PR

File: frontend/src/components/templates/GuideTemplate/GuideTemplate.module.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+    background-color: var(--background-color);
+    box-sizing: border-box;
+    padding: 10px 150px 10px 70px;
+    width: 100%;
+
+    .popup {
+        height: 100%;
+        width: 100%;
+        background-color: white;
```
Comment (uparkalau) [inline frontend/src/components/templates/GuideTemplate/GuideTemplate.module.scss:13]: what is this colors?

File: frontend/src/scenes/bannerPage/BannerPage.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                            isTopPosition={isTopPosition}
+                            bannerText={bannerText}
+                            setBannerText={setBannerText}
+                        />}
+                    leftContent={() =>
+                        <BannerLeftContent
+                            setIsTopPosition={setIsTopPosition}
+                        />}
+                    leftAppearance={() => (
+                        <BannerLeftAppearance
```
Comment (uparkalau) [inline frontend/src/scenes/bannerPage/BannerPage.jsx:39]: why this is only for banner page can we reuse it for other pages?

File: frontend/src/components/templates/GuideTemplate/GuideTemplate.module.scss [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        box-shadow: 0px 8px 8px -4px #10182808;
+        box-shadow: 0px 20px 24px -4px #10182814;
+
+        .header {
+            font-size: 20px;
+            font-weight: bold;
+            padding: 20px;
+            background-color: #F8F9F8;
```
Comment (uparkalau) [inline frontend/src/components/templates/GuideTemplate/GuideTemplate.module.scss:21]: if it is ne color add it into variables

File: frontend/src/components/BannerPageComponents/BannerLeftAppearance/BannerLeftApperance.jsx
Comment (uparkalau) [inline frontend/src/components/BannerPageComponents/BannerLeftAppearance/BannerLeftApperance.jsx:1]: can we reuse this component for other pages? If yes rename and refactor it the others can use it.

File: frontend/src/components/BannerPageComponents/BannerLeftContent/BannerLeftContent.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+            <div className={styles.radioContent}>
+                <Radio onClick={() =>setIsTopPosition(true)}/>
+                <h3>Top (centered)</h3>
+            </div>
+            <div className={styles.radioContent}>
+                <Radio onClick={() =>setIsTopPosition(false)}/>
+                <h3>Bottom (centered)</h3>
+            </div>
+            <h2 style={{marginBottom:0}}>URL</h2>
+            <CustomTextField TextFieldWidth='241px'/>
```
Comment (uparkalau) [inline frontend/src/components/BannerPageComponents/BannerLeftContent/BannerLeftContent.jsx:22]: why it 241px. i think it has to be flexible

**Review by uparkalau — APPROVED** [review]
please implement changes where possible. Overall it looks good just need a little refactoring. Refactor what you can and merge.

File: frontend/src/scenes/bannerPage/BannerPage.jsx (hunk 2 above)
Comment (erenfn) [reply frontend/src/scenes/bannerPage/BannerPage.jsx:39]: This is only for the banner page. Popup page is similar but it is just better to create a new one for it

File: frontend/src/components/templates/GuideTemplate/GuideTemplate.module.scss (hunk 1 above)
Comment (erenfn) [reply frontend/src/components/templates/GuideTemplate/GuideTemplate.module.scss:13]: I just copied them from the figma file.

**Review by uparkalau — APPROVED** [review]

**Review by uparkalau — APPROVED** [review]

**Pull request merged**